rust/protocol/src/sealed_sender.rs

```diff
@@ -565,11 +565,13 @@ impl UnidentifiedSenderMessage {
             SEALED_SENDER_V2_VERSION => {
                 // Uses a flat representation: C || AT || E.pub || ciphertext
                 let remaining = &data[1..];
-                if remaining.len() < 32 + 16 + 32 {
+                if remaining.len() < sealed_sender_v2::M_LEN + sealed_sender_v2::AT_LEN + 32 {
```

Comment:
nit: Should the 32 also be a const, or is it well-known? It stands out in the diff.

Reply:
:-/ The constant for it is in another module and isn't currently public, but you're right, it should be.